resources/js/stores/useCart.js

```diff
@@ -45,6 +46,7 @@ export const clear = async function () {
     await clearAddresses()
     await clearMask()
     await refresh()
+    cartStorage.value = {}
```

Comment:
Are we sure we needs this? As `refresh()` is called, if the mask/token isn't there it will be cleared; otherwise a request will be made and the storage will be updated.

Reply:
In this case the token will still be there when the customer is logged in.
So it will trigger the 404 and empty the cart after the request finishes, this 404 is intended as we do not know wether the user has a cart